Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "b5ee7f98f76daedb5e5b92e8d12ba8a8e0bf8890f4db84deba904cfe0dd7e20e"
+            "sha256": "1715c4c8cb6049cbbb98afcc0f5a885ebd2c9b6f5df357e83c20fbd860472c01"
```

Comment:
When I run `./dev-tools/install --clean`, the `Pipfile.lock` is changed again... Doesn't this happen on your machine?
In theory, we have a circleci check for this:
https://github.com/digitalfabrik/integreat-cms/blob/534f226f80618be350bb10c1e6d7c7448147db66/.circleci/config.yml#L23-L27

But somehow, this does not seem to work here :thinking:
Probably, because the old venv state is cached based on the `Pipfile.lock` hash (which didn't change obviously)...

Anyways, could you commit the diff in the `Pipfile.lock` as well?

Reply:
Sure. Did that. No idea though what happened and why? :D

Reply:
I think for pipenv it makes a difference whether a package is a direct dependency in the Pipfile, or an implicit dependency by some package in the Pipfile (in this case `integreat-cms`)... After you moved the dependency into `setup.cfg`, the lock file needed to be regenerated...